SpatialGDK/Documentation/content/directory-structure.md

```diff
@@ -1,22 +1,21 @@
 <%(Callout type="warn" message="This [pre-alpha](https://docs.improbable.io/reference/latest/shared/release-policy#maturity-stages) release of the SpatialOS GDK for Unreal is for evaluation and feedback purposes only, with limited documentation - see the guidance on [Recommended use]({{urlRoot}}/index#recommended-use)")%>
 
 # GDK for Unreal directory structure
-The table below contains the contents of the built GDK for Unreal after running `BuildGDK.bat`.
+The table below contains the contents of the Unreal GDk repo folder after running `Setup.bat`.
 
 | Directory | Purpose
 |-----------|---------
-| `Binaries/ThirdParty/Improbable/` | (Not tracked in git) This folder contains all binaries required for your game project with the GDK for Unreal. These files are generated by `BuildGDK.bat`.
-| `build/` | (Not tracked in git) Intermediate folder used for build steps while running `BuildGDK.bat`.
-| `linting/` | Contains the linting scripts for checking the GDK for Unreal.
-| `docs` | Contains the documentation for the GDK for Unreal.
-| `fastbuild` | Contains files related to installing FASTBuild. **Note** these currently only work for Improbable engineers.
-| `packages/` | (Not tracked in git) Contains the dependencies for building the Unreal SDK codegen.
-| `Plugins/SpatialGDK/SpatialGDKEditorToolbar/` | The [SpatialOS GDK for Unreal toolbar]({{urlRoot}}/content/toolbar.md) within the Unreal Editor, from which you can take snapshots, generate interop code, start and stop deployments, and access the SpatialOS Inspector.
-| `schema` | Contains the schema files required for the GDK to interact with SpatialOS.
-| `Scripts/` | Contains a set of helper scripts used to build a GDK worker.
-| `Source/Programs/` | Contains the Unreal SDK code generator.
-| `Source/SpatialGDK/Legacy/` | Contains the source code from the [GitHub-hosted Unreal SDK](https://github.com/spatialos/UnrealSDK/tree/master/Game/Source/SpatialOS).
-| `Source/SpatialGDK/Legacy/Deprecated/` | Contains the forked source code from the [GitHub-hosted Unreal SDK](https://github.com/spatialos/UnrealSDK/tree/master/Game/Source/SpatialOS) that the GDK does not depend on. This is present purely for backwards compatibility.
-| `Source/SpatialGDK/Public/WorkerSdk/` | (Not tracked in git) Contains the [C++ worker SDK](https://docs.improbable.io/reference/latest/cppsdk/introduction) headers. You install these when you run `BuildGDK.bat`
-| `Source/SpatialGDK/Public` | Public source code of the GDK for Unreal module.
-| `Source/SpatialGDK/Private` | Private source code of the GDK for Unreal module.
+| `SpatialGDK/Binaries/ThirdParty/Improbable/` | (Not tracked in git) This folder contains all binaries required for your game project with the GDK for Unreal. These files are generated by `BuildGDK.bat`.
```

Comment:
I would prefer that you remove

"(Not tracked in git)"

and replace it with a line at the bottom of the table cell that says

"This directory is not tracked by git."